pollination/honeybee_radiance/multiphase.py

```diff
@@ -56,3 +56,48 @@ def run_view_mtx(self):
     view_mtx = Outputs.file(
         description='Output view matrix file.', path='output.vmx'
     )
+
+
+@dataclass
+class DaylightMatrix(Function):
+    """Calculate daylight matrix for a sender file."""
+
+    radiance_parameters = Inputs.str(
+        description='Radiance parameters. -aa 0 is already included in '
+        'the command.', default=''
+    )
+
+    fixed_radiance_parameters = Inputs.str(
+        description='Radiance parameters. -aa 0 is already included in '
+        'the command.', default='-aa 0'
+    )
+
+    sender_file = Inputs.file(
+        description='Path to a sender file.', path='sender.rad',
+        extensions=['rad']
+    )
+
+    receiver_file = Inputs.file(
+        description='Path to a receiver file.', path='receiver.rad',
+        extensions=['rad']
+    )
```

Comment:
Isn't the receiver file always a sky? Do we need it as an input here? Or should we generate the sky under the hood?

Reply:
@mostaphaRoudsari, since we renamed the honeybee-radiance command to "flux-transfer" would it better to do the same with this function, i.e. "FluxTransfer"? This will give a generic flux transfer function, that can be used for any rfluxmtx run, including daylight matrices. Or is it better to keep the function specifically for daylight matrices?

Reply:
Hi @mikkelkp! I think that's a good idea. We should probably also update the command in this function.